editor/svgcanvas.js

```diff
@@ -4446,6 +4446,8 @@ this.setSvgString = function(xmlString) {
 	try {
 		// convert string into XML document
 		var newDoc = svgedit.utilities.text2xml(xmlString);
+		if(newDoc.firstChild.namespaceURI != 'http://www.w3.org/2000/svg')
+			return false;
```

Comment:
`firstChild` should probably be `firstElementChild`, or rather, use jQuery for backward-compatibility, to ensure it is referencing the first element and not erring by trying to access a namespace on a comment node or processing instruction.